tests/apollo/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ if(LEAKCHECK)
   # For Leak Sanitizers to report errors we need to shut down processes gracefully
   set(APOLLO_TEST_ENV "${APOLLO_TEST_ENV} GRACEFUL_SHUTDOWN=true")
   set(APOLLO_TEST_ENV "${APOLLO_TEST_ENV} \
-  ASAN_OPTIONS=log_path=${ASAN_LOG_DIR}/leak.log:symbolize=1:check_initialization_order=1:detect_leaks=1 \
+  ASAN_OPTIONS=log_path=${ASAN_LOG_DIR}/asan.log:symbolize=1:check_initialization_order=1:detect_leaks=1 \
```

Comment:
Not sure I understand. This ist he default full path (e.g /concord-bft/build/asan_logs)

Reply:
have a ${ASAN_LOG_FILE_PATH} environment variable instead of ${ASAN_LOG_DIR}/leak.log
this gives more flexibility to change the file name/location in the future (for example, having replica ids in the file name).

Reply:
This is another improvment, I'm sorry but I won't be able to do it now as I have no time capacity, This is quite complicated in the code and increase work time in hours/days.
openning a new task : https://jira.eng.vmware.com/browse/BC-26378